documentation/user-interface/report-view.md

```diff
@@ -22,11 +22,11 @@ The **Report** view displays a detailed report about a test run. It allows you t
 
 ## Analyze Test Results
 
-The **Report** view can present data in different formats. They include machine-readable formats like JSON or formats that better fit for human examination like a grid view. Use a drop-down list in the report header to select a format.
+The **Report** view can display data in different formats. You can use the report header's drop-down list to select a format.

```

Comment:
The whole Analyze Test Results section is too wordy. You can combine and reorder the information in this section to improve the readability and reduce the number of words. The part about 'better fit for human examination' is strange, and the image already shows the available formats.

Reply:
> The whole Analyze Test Results section is too wordy. You can combine and reorder the information in this section to improve the readability and reduce the number of words. The part about 'better fit for human examination' is strange...

okay, I see

> the image already shows the available formats.

well, I don't treat images as a reliable source of information. They should be used only for illustrative purposes. In this case, showing where the drop-down can be invoked and how it looks like, While important information like the list of available report formats should be written in the text (even if it duplicates the screenshots).